### DISCUSSION TITLE
How to setup second inverter (which also has 192.168.200.1 IP address)?

### DISCUSSION THREAD

**Discussion by stefanhapper** [wlcrs/huawei_solar#513] [Connecting to the inverter]
I just got my second Huawei 2000 4.6 KTL L1 inverter. I have two separate USB Wifi adapters on my Home Assistant server and these are connected to the two WIFI access points from the two inverters.

I also managed to setup a second "Huawei Solar" integration and connect to the second inverter. However, the data now gets mixed up as both inverters have the 192.168.200.1 IP address.

Any idea on how I can get this setup to work? I don't want to use cascading.

**Comment by Roving-Ronin** [comment]
@stefanhapper are these two x 4kW L1 (single phase) inverters installed next to each other, providing power into the same home / electrical network?

Assuming so....

You need to be looking to pick one as the primary inverter and then get the RS485 (serial cable) linked from the primary inverter to the secondary inverter (the DDSU-666H meter should already be connected to the RS485 of the primary  / original inverter. You also only have a single sDongle (WiFi/FE or 4G) in use for the overall setup, that is also connected into the primary inverter.

Suggest reading the user / installation manuals: https://support.huawei.com/enterprise/en/doc/EDOC1100136173/daff17bb/optional-connecting-signal-cables

Once you have this properly installed (and setup in FusionSolar also, so it is a single 'plant' that contains your 2 inverters and sub dongle) then you can use this integration. With your usb sdongle connected to your home WiFi / LAN, you can then connect it its IP (and modbus tcp port) and gain access to the primary inverter AND the secondary inverter (that the primary passes the info on from/to).
See: https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter#connect-to-the-modbus-tcp-port-exposed-by-your-dongle-on-your-home-network

**Reply by stefanhapper** [reply]
Hi @Roving-Ronin, many thanks for your reply. It is indeed two single-phase converters next to each other.

I wanted to avoid cascading them through a RS485 cable and having to get the dongle. I could not work it out to connect both via the the access point WIFI but finally I managed to have one via the access point WIFI and the other one with the direct wifi connection via the home network (having the O&M option enabled first in the FusionSolar app).

The setup now works great: I have full access to both inverters's data and can add them up manually in HA where I want.
